library/ws-widget/collection/src/lib/card-home-discuss/card-home-discuss.component.scss

```diff
@@ -152,4 +169,27 @@
 
 .mat-card-footer {
   padding-bottom: 22px;
+  margin-left: -18px;
+}
+
+.mat-card-sub-title-marign
```

Comment:
Dont override the common classes and styles

Reply:
If u have to override then use one more custom class name and then give additional styles